examples/shared_samples/interconnect.rb

```diff
@@ -15,16 +15,10 @@
 # NOTE: You'll need to add an interconnect with state Configured and a port linked.
 #
 # Supported APIs:
-# - 200, 300, 500, 600
-#
-# Resources that can be created according to parameters:
-# api_version = 200 & variant = any to OneviewSDK::API200::Interconnect'
-# api_version = 300 & variant = C7000 to OneviewSDK::API300::C7000::Interconnect'
-# api_version = 300 & variant = Synergy to OneviewSDK::API300::Synergy::Interconnect'
-# api_version = 500 & variant = C7000 to OneviewSDK::API500::C7000::Interconnect'
-# api_version = 500 & variant = Synergy to OneviewSDK::API500::Synergy::Interconnect'
-# api_version = 600 & variant = C7000 to OneviewSDK::API600::C7000::Interconnect'
-# api_version = 600 & variant = Synergy to OneviewSDK::API600::Synergy::Interconnect'
+# - 200, 300, 500, 600, 800, 1000, 1200, 1600
+# Supoported Variants
+# C7000, Synergy
```

Comment:
Not sure why parameter details were removed. Need to add them.

Reply:
done